exposee-app/server.js

```diff
@@ -113,6 +127,34 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
+app.post('/broadcast',verifytoken, async (req, res)=> {
+  try {
+    const {url, description, duration, api_key} = req.body;
+    const userId = req.userId
+    if (!url|| !description|| !duration || !userId||!api_key)
+    {
+     return res.status(400).json({error: 'all fields are required'})
+    }
+    const user = await User.findByPk(userId);
+    if (!user){
+      return res.status(404).json({error: 'user not found'});
+    }
+    const video = await Video.create({
+      title: "STREAM",
+      url,
+      description,
+      userId,
```

Comment:
sorry can we change the model on this to be user_id
and make sure all columns are consistently named?